stdlib/3/tkinter/__init__.pyi

```diff
@@ -274,31 +284,44 @@ class Misc:
     def nametowidget(self, name): ...
     register: Any
     def configure(self, cnf: Optional[Any] = ..., **kw): ...
-    config: Any
+    config = configure
     def cget(self, key): ...
     __getitem__: Any
     def __setitem__(self, key, value): ...
     def keys(self): ...
-    def pack_propagate(self, flag=...): ...
-    propagate: Any
-    def pack_slaves(self): ...
-    slaves: Any
-    def place_slaves(self): ...
-    def grid_anchor(self, anchor: Optional[Any] = ...): ...
-    anchor: Any
+    @overload
+    def pack_propagate(self, flag: bool) -> Optional[bool]: ...
+    @overload
+    def pack_propagate(self, flag: List[Literal["_noarg_"]] = ...) -> None: ...
+    propagate = pack_propagate
+    def grid_anchor(self, anchor: Optional[_Anchor] = ...) -> None: ...
+    anchor = grid_anchor
+    @overload
     def grid_bbox(
-        self, column: Optional[Any] = ..., row: Optional[Any] = ..., col2: Optional[Any] = ..., row2: Optional[Any] = ...
-    ): ...
-    bbox: Any
-    def grid_columnconfigure(self, index, cnf=..., **kw): ...
-    columnconfigure: Any
-    def grid_location(self, x, y): ...
-    def grid_propagate(self, flag=...): ...
-    def grid_rowconfigure(self, index, cnf=..., **kw): ...
-    rowconfigure: Any
-    def grid_size(self): ...
-    size: Any
-    def grid_slaves(self, row: Optional[Any] = ..., column: Optional[Any] = ...): ...
+        self, column: None = ..., row: None = ..., col2: None = ..., row2: None = ...
+    ) -> Optional[Tuple[int, int, int, int]]: ...
+    @overload
+    def grid_bbox(self, column: int, row: int, col2: None = ..., row2: None = ...) -> Optional[Tuple[int, int, int, int]]: ...
+    @overload
+    def grid_bbox(self, column: int, row: int, col2: int, row2: int) -> Optional[Tuple[int, int, int, int]]: ...
```

Comment:
Just an observation: Technically, `grid_bbox(col2=123, row2=321)` is possible, but it behaves exactly like `grid_bbox(column=123, row=321)`, which is quite confusing. Therefore not having that overload is probably the better choice.